src/components/scenes/RequestScene.tsx

```diff
@@ -538,6 +550,12 @@ const getStyles = cacheStyles((theme: Theme) => ({
     alignSelf: 'flex-end',
     fontFamily: theme.fontFaceBold
   },
+  qrContainer: {
+    margin: theme.rem(0.5),
+    alignSelf: 'center',
+    flex: 1,
```

Comment:
Though `flex: 1` works here, prefer to be explicit per our flex housekeeping:

```suggestion
    flexGrow: 1,
    flexShrink: 1,
```